src/Blog/templates/list.phtml

```diff
@@ -1,7 +1,13 @@
 <?php
-$this->layout('layout::default', [
-    'title' => 'Blog',
-]);
+
+declare(strict_types=1);
+
+/**
+ * @var League\Plates\Template\Template $this
```

Comment:
Plates? 🤔

Reply:
https://github.com/laminas/getlaminas.org/blob/72a84f221c7c5f40ffac00bcddf6f902064641ba/composer.json#L55

Reply:
I confused this repo with a general-purpose example: my bad :-)